feature-libs/customer-ticketing/styles/components/_customer-ticketing-actions.scss

```diff
@@ -1,16 +1,18 @@
-%cx-customer-ticketing-close,
-%cx-customer-ticketing-reopen {
-  display: flex;
-  justify-content: flex-end;
+%cx-customer-ticketing-add {
```

Comment:
what is `cx-customer-ticketing-add` as I don't see any component with that selector 

Moreover, it is wrapping the other placeholder selectors, which I don't think make sense. I believe this was changed accidentally?

Reply:
I believe it was intended for the -create component

Reply:
I believe this was added with the intention of adding the -create component button.

Maybe it's better to create your own stylesheet and extending one of these as I don't think this button should be taking the full width of the container as shown in the wirefrrames

![Screen Shot 2022-08-25 at 3 33 16 PM](https://user-images.githubusercontent.com/29099466/186760086-35c928bd-a274-44b4-be81-0f1d52114542.png)

Reply:
This is fixed with https://github.com/SAP/spartacus/pull/16063/commits/cfc3a2fd847515d99677e7b566ee3fa7b3e000a5